src/Makefile.am

```diff
@@ -708,6 +709,7 @@ libsuricata_c_a_SOURCES = \
 	detect-engine-alert.c \
 	detect-engine-analyzer.c \
 	detect-engine-build.c \
+	detect-engine-buffer.c \
```

Comment:
nit: out of order, but I'd only fix on a next rebase or something

Reply:
Argh my bad